jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/startup/CodeWhispererProjectStartupActivity.kt

```diff
@@ -113,6 +123,16 @@ class CodeWhispererProjectStartupActivity : StartupActivity.DumbAware {
             parsedLastShown.plusDays(7) <= LocalDateTime.now()
         } ?: true
     }
+
+    // Start a job that runs every 30 mins
+    private fun initFeatureConfigPollingJob(project: Project) {
+        projectCoroutineScope(project).launch {
+            while (isActive) {
+                CodeWhispererFeatureConfigService.getInstance().fetchFeatureConfigs(project)
```

Comment:
given it's a app level config, shouldn't we put this part of logic in a app level component instead?
think this way the network overhead could be slightly reduced

Reply:
nm, either way we will still need "project" to check connection validity. But we need some debounce logic maybe.